.github/workflows/build-test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     uses: playframework/.github/.github/workflows/cmd.yml@v3
     with:
       java: 17, 11
-      scala: 2.12.17, 2.13.8
+      scala: 2.13.8
```

Comment:
How about already use `2.13.x` like in Play JSON (https://github.com/playframework/play-json/pull/794)?

Reply:
Doing that right now :wink: